src/platform/plugins/shared/content_management/common/rpc/rpc.ts

```diff
@@ -17,14 +17,14 @@ import { deleteSchemas } from './delete';
 import { searchSchemas } from './search';
 import { mSearchSchemas } from './msearch';
 
-export const schemas: {
-  [key in ProcedureName]: ProcedureSchemas;
-} = {
+export const schemas = {
```

Comment:
The same applies here, if we use `satisfies` we can then use `schemas.get` and see the actual schema definition types.